kdcount/correlate.py

```diff
@@ -761,7 +761,7 @@ def work(self, i):
 
             counts, sum1c = n1.count(n2, self.bins.edges,
                                 attrs=sum1attrs)
-            sum1[..., :-1] = sum1c
+            sum1[..., :-1] = sum1c.T
```

Comment:
@rainwoodman Please can you review this? 

The error without it is:
ValueError: could not broadcast input array from shape (10,1) into shape (1,10)
I *think* that the count method always returned (10,1) here, but old numpy broadcasting rules allowed it to work anyway. But I would like you to confirm.